ipyleaflet/leaflet.py

```diff
@@ -782,6 +782,79 @@ class VideoOverlay(RasterLayer):
     attribution = Unicode().tag(sync=True, o=True)
 
 
+class ImageService(Layer):
+    """ImageService class
+
+    Image Service layer for raster data served through a web service
+
+    Attributes
+    ----------
+    url: string, default ""
+        Url to the image service
+    f: string, default "image"
+        response format (use `'image'` to stream as bytes)
+    format: string, default "jpgpng"
+        format of exported image
+    pixel_type: string, default "UNKNOWN"
+        data type of the raster image
+    no_data: list, default []
+        pixel value or comma-delimited list of pixel values representing no data
```

Comment:
In Python, elements of a list are always comma-delimited, so this is not worth mentioning.
If it can be a pixel value, then the type should reflect that (currently it has to be a `list`).

Reply:
great point. Edited to specify `list` only